features/white-space-collapse.yml

```diff
@@ -0,0 +1,10 @@
+name: white-space-collapse
+description: The `white-space-collapse` CSS property sets whether new line characters are shown as line breaks, and whether multiple consecutive spaces are all displayed or combined.
+spec: https://drafts.csswg.org/css-text-4/#white-space-collapsing
+group: white-space
+compat_features:
```

Comment:
There are several values defined in the spec that are not listed here (and don't have BCD keys) - such as `preserve-spaces` and `discard`. What's the proper way to handle that?

Reply:
Anything that's implemented in at least one browser ought to have a key in BCD, so we're definitely missing at least `preserve-spaces`, perhaps more. I've filed https://github.com/mdn/browser-compat-data/issues/24363 about this case specifically.

Reply:
For largely historical reasons, BCD didn't record CSS property values comprehensively, only values added "later" (subjective, sometimes). I brought this up in a BCD meeting recently and was told to file issues about missing values.